pkg/scraper/mapped.go

```diff
@@ -276,11 +278,33 @@ func (p *postProcessMap) Apply(value string, q mappedQuery) string {
 	return value
 }
 
+type postProcessToMetric bool
+
+func (p *postProcessToMetric) Apply(value string, q mappedQuery) string {
+	const foot_in_cm = 30.48
+	const inch_in_cm = 2.54
+
+	reg := regexp.MustCompile("[0-9]+")
+	filtered := reg.FindAllString(value, -1)
+
+	var parsed []float64
+	for _, v := range filtered {
+		k, _ := strconv.ParseFloat(v, 64)
+		parsed = append(parsed, k)
+	}
+
+	var centimeters = parsed[0]*foot_in_cm + parsed[1]*inch_in_cm
```

Comment:
This is going to panic if the input only has a single number or doesn't have numbers at all.

```suggestion
	var feet float64
	var inches float64
	if len(filtered) > 0 {
		feet, _ = strconv.ParseFloat(filtered[0], 64)
	}
	if len(filtered) > 1 {
		inches, _ = strconv.ParseFloat(filtered[1], 64)
	}

	centimeters := feet*foot_in_cm + inches*inch_in_cm
```

Reply:
Thanks for the patch, this is indeed more safe.